modules/ROOT/pages/op-installing-tkn-on-windows.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="installing-tkn-on-windows"]
+
+= Installing {op-title} on Windows
+
+== Binary installation
+
+. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/tkn/latest/tkn-windows-amd64.exe[`tkn.exe`] file.
+. Add the location of your `tkn.exe` to the `PATH` variable.
+
+[discrete]
+== Setting the `PATH` variable for Windows 7/8
```

Comment:
Question: I know odo docs have these 2 sections, but do you think we need to add this information? IMHO, things could change for future versions and we will need to keep updating this. Also, do you think it would be safe to think that a windows developer might know how to set the `PATH` variable? WDYT? Let's discuss and probably make it an uniform convention for all the DevTools docs.

Reply:
I think since our audience are developers, we can safely assume that they would be aware of the PATH environment variable, and should be able to set it up. As you rightly pointed out, it would be a maintenance overhead for us, in case newer versions of windows come in later.